src/main/kotlin/no/nav/familie/ef/mottak/service/DbMigreringService.kt

```diff
@@ -0,0 +1,47 @@
+package no.nav.familie.ef.mottak.service
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.ef.mottak.repository.SoknadRepository
+import no.nav.familie.ef.mottak.repository.VedleggRepository
+import no.nav.familie.ef.mottak.repository.domain.Fil
+import no.nav.familie.ef.mottak.repository.domain.Soknad
+import no.nav.familie.kontrakter.ef.søknad.Vedlegg
+import no.nav.familie.kontrakter.felles.objectMapper
+import org.slf4j.LoggerFactory
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Service
+import javax.transaction.Transactional
+import no.nav.familie.ef.mottak.repository.domain.Vedlegg as DbVedlegg
+
+@Service
+class DbMigreringService(private val soknadRepository: SoknadRepository,
+                         private val vedleggRepository: VedleggRepository) {
+
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    @Scheduled(initialDelay = 10000, fixedDelay = 60000)
+    @Transactional
+    fun dbMigrering() {
+        val soknad = soknadRepository.findFirstByVedleggIsNotNull()
```

Comment:
Vi riskierer ikke att denne kjører samtidig som en annen jobb? Og att 2 jobb oppdaterer en søknad samtidig?

Reply:
Transaksjonen skal sørge for at det ikke skjer.